snappy-tools/sbin/snappy-start-all.sh

```diff
@@ -34,4 +34,6 @@ sbin="`cd "$sbin"; pwd`"
 "$sbin"/snappy-servers.sh start
 
 # Start Leads
-"$sbin"/snappy-leads.sh start
+if [ "$1" != "rowstore" ]; then
+  "$sbin"/snappy-leads.sh start
```

Comment:
For stopping if we maintain that the cluster was started with rowstore option then skip stopping the lead node.